.circleci/config.yml

```diff
@@ -340,9 +340,13 @@ workflows:
 
       # Release
       - horizon/block:
-          <<: *only_release
           context: horizon
           project_id: 11
+          filters:
+            branches:
+              only:
+                - release
+                - staging
```

Comment:
To make sure I understand, this is a test to see if the deploy PR would get a "X" when staging build fails, right?

Reply:
Yes exactly. I'm just reusing the existing `horizon/block` step for that purpose, since it's already in place and we can easily create a block temporarily to test how it works. (I expect the eventual step we'll want to add will require checking out the Integrity project and other complicating factors that I didn't want to attempt before this validation.)